pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[build-system]
+requires = ["setuptools>=40.9.0", "wheel", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pip-licenses"
+description = "Dump the software license list of Python packages installed with pip."
+dynamic = ["version", "readme"]
+requires-python = ">=3.8"
+license = {file = "LICENSE"}
+authors = [
+    {name = "raimon", email = "raimon49@hotmail.com"}
+]
+keywords = ["pip", "pypi", "package", "license", "check"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: System :: Systems Administration",
+    "Topic :: System :: System Shells",
+    "Typing :: Typed"
+]
+dependencies = [
+    "prettytable >= 2.3.0",
+    "tomli >= 2"
+]
+
+[project.optional-dependencies]
+dev = [
+    "autopep8",
+    "black",
+    "docutils",
+    "isort",
+    "wheel",
+    "mypy",
+    "pip-tools",
+    "pypandoc",
+    "pytest-cov",
+    "pytest-pycodestyle",
+    "pytest-runner",
+    "twine",
+    "tomli-w"
+]
+
+[project.urls]
+Homepage = "https://github.com/raimon49/pip-licenses"
+Releases = "https://github.com/raimon49/pip-licenses/releases"
+Issues = "https://github.com/raimon49/pip-licenses/issues"
+
+[project.scripts]
+pip-licenses = "piplicenses:main"
+
+[tool.setuptools]
+py-modules = ["piplicenses"]
+
+[tool.setuptools.dynamic]
+version = {attr = "piplicenses.__version__"}
+readme = {file = ["README.md", "CHANGELOG.md"]}
```

Comment:
@mbalatsko Thanks for finding a way to combine the README documents.

I tried uploading to TestPyPI and it returned an error regarding file content type.

```bash
ERROR    HTTPError: 400 Bad Request from https://test.pypi.org/legacy/
         The description failed to render for 'text/x-rst'. See https://test.pypi.org/help/#description-content-type for
         more information.
```

Don't we need a `content-type` key along with the `file` key?

```toml
readme = {file = ["README.md", "CHANGELOG.md"], content-type = "text/markdown"}
```

Reply:
Yeah, in case we see that error. I was under intention that it is default, let me push fix for it now

Reply:
@raimon49 pushed, could you try it out please?

Reply:
Good, error resolved 👍 
https://test.pypi.org/project/pip-licenses/4.6.0rc1/

One thing I saw and noticed is that the license metadata is very long full text when packaged with `license = {file = “LICENSE”}`.

Wouldn't the pip-licenses command create a very large table in the output? Of course, Classifiers take precedence by default, so it is not worth worrying about.

Did you change from text to file because file is the preferred method?
b05111ee24443388c028fed2bdf3b5cbc52287a0

Reply:
According to https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license both options are equally right. I just wanted to make use of file we have in repository, no other reason. 
I tried to run `pip-licenses --with-system --from meta`:
```
 pip-licenses        4.5.1            MIT License                                                                            
                                                                                                                             
                                              Copyright (c) 2018 raimon                                                      
                                                                                                                             
                                              Permission is hereby granted, free of charge, to any person obtaining a copy   
                                              of this software and associated documentation files (the "Software"), to deal  
                                              in the Software without restriction, including without limitation the rights   
                                              to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      
                                              copies of the Software, and to permit persons to whom the Software is          
                                              furnished to do so, subject to the following conditions:                       
                                                                                                                             
                                              The above copyright notice and this permission notice shall be included in all 
                                              copies or substantial portions of the Software.                                
                                                                                                                             
                                              THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     
                                              IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       
                                              FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    
                                              AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         
                                              LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  
                                              OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  
                                              SOFTWARE.     
```

And it is indeed long, but if you use mixed `pip-licenses --with-system --mixed`:

```
 pip-licenses        4.5.1            MIT License 
```

Reply:
If you'd prefer to use textual format, I have no problems fixing it in the PR. But I suggest using `MIT License` instead of `MIT` as it is more semantically correct

Reply:
Thanks for the info. It still looks like I should adopt `text`.

The notation `MIT` also seems to be a short identifier standardized by SPDX.
https://spdx.org/licenses/

However, your suggestion is also acceptable.

Can you change the `text` to either full or short and push it? I will merge it and ship as a new release.